tests/test_launcher.py

```diff
@@ -47,6 +65,18 @@ def test_additional_argument_g_gu():
 
 
 def test_container_launcher():
+    if not check_docker_support():
+        error_message = (
+            "Docker is not working correctly in this system, "
+            "yet a Fluent Docker container launch was specified."
+        )
+        with pytest.raises(DockerContainerLaunchNotSupported) as msg:
+            container_dict_1 = pyfluent.launch_fluent(start_container=True)
+            container_dict_2 = pyfluent.launch_fluent(
+                start_container=True, dry_run=True
+            )
+        assert msg.value.args[0] == error_message
```

Comment:
The message is whatever is in DockerContainerLaunchNotSupported. This part isn't really testing anything. It's enough to test that an exception of the specific type is caught.

Reply:
Yes will raise and catch exception only, we will not verify it's error message. I just followed existing testing format.

Reply:
Done. Thank you.